livekit-agents/livekit/agents/voice/avatar/_datastream_io.py

```diff
@@ -108,11 +110,18 @@ class DataStreamAudioReceiver(AudioReceiver):
     subscribe to the first agent participant in the room.
     """
 
-    def __init__(self, room: rtc.Room, *, sender_identity: str | None = None):
+    def __init__(
+        self,
+        room: rtc.Room,
+        *,
+        sender_identity: str | None = None,
+        duration_per_frame: NotGivenOr[float] = NOT_GIVEN,
```

Comment:
In other parts of the code, we always do https://github.com/livekit/agents/blob/d30d7c13b3591b94f4be59574dc2fb6fee81cede/livekit-agents/livekit/agents/utils/audio.py#L58
https://github.com/livekit/agents/blob/d30d7c13b3591b94f4be59574dc2fb6fee81cede/livekit-agents/livekit/agents/tts/tts.py#L543

Can we keep names consistent? (one of them)